blockchain/vm/contracts.go

```diff
@@ -60,9 +60,9 @@ type PrecompiledContract interface {
 	Run(input []byte, contract *Contract, evm *EVM) ([]byte, error)
 }
 
-// PrecompiledContractsCypress contains the default set of pre-compiled Klaytn
-// contracts used in the Cypress release.
-var PrecompiledContractsCypress = map[common.Address]PrecompiledContract{
+// PrecompiledContractsConstantinople contains the default set of pre-compiled Klaytn
+// contracts used in the Constantinople release.
```

Comment:
I though `Constantinople release` is strange.
How about this?

```suggestion
// PrecompiledContractsConstantinople contains the set of pre-compiled Klaytn
// contracts based on Ethereum Constantinople.
```

Reply:
Good idea! I'll reflect it.